app/code/Magento/Wishlist/Block/Adminhtml/WishlistTab.php

```diff
@@ -42,7 +42,7 @@ public function __construct(Context $context, Registry $registry, array $data =
     }
 
     /**
-     * @inheritdoc
```

Comment:
Hi @NishaJangid. Thanks for collaboration. ```@inheritdoc``` annotation means that function already declared in parent scope or interface and you can see the method annotation theirs. It means that not needed to declare type annotation directly for this method.